Sources/StreamChat/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -209,6 +211,72 @@ class ChannelDTO_Tests: XCTestCase {
         XCTAssertEqual(channel?.latestMessages.count, 10)
     }
 
+    func test_channelPayload_pinnedMessagesOlderThanOldestMessageAtAreIgnored() throws {
+        // Save a channel payload with 100 messages
```

Comment:
that `100` there is probably a copy/paste mistake? Also, why even use 20? Wouldn't just 1+1 be better and more obvious? :)

Reply:
Yes copy paste issue 😄 Yeah you right, 1+1 will be easier to understand what we are testing here, will change it 👍